Update package with fixes to constant imports

chalenascholl opened the pull request:
1) Updates the package to import the necessary constants in the correct manner, as the `src` folder is [not exposed](https://www.npmjs.com/package/@mapbox/mapbox-gl-draw?activeTab=code) in the `mapbox-gl-draw` package

2) Include `dist` folder for now, so that others can use this repo, until we can hopefully get this change into the original package and published to npm as well.

chalenascholl committed:
update packages and imports from mapbox-gl-draw

chalenascholl committed:
add dist to repo so it can be used directly, since we can't publish a new package

chalenascholl committed:
add draw package as dev dependency

chalenascholl committed:
don't need to change import here

chalenascholl committed:
update readme with correct way to import

chalenascholl committed:
update demo with correct import

chalenascholl commented:
To use these changes instead of the original package, link to the github repo with these changes like so:

`"mapbox-gl-draw-split-polygon-mode": "https://github.com/chalenascholl/mapbox-gl-draw-split-polygon-mode.git",`

chalenascholl commented:
@ReyhaneMasumi or @mhsattarian thanks for creating this custom mode! Are either of you able to review these changes?

crawld commented:
@chalenascholl thanks for the update to fix the imports. when i use your version there are no errors, but i also don't see the button to split polygons on the map. in the demo video, the button to split polygons has a scissors icon; where is this defined?

chalenascholl committed:
add changes to make demo page work again

chalenascholl committed:
update readme to include adding the button to the draw controls

chalenascholl committed:
update Readme to link the relevant info for custom control

chalenascholl committed:
slight word changes in the readme

chalenascholl commented:
@crawld sorry for the very long delay on the response! It had been a while since I used it myself so I needed to refresh my memory and finally made some time now.

The consumer of the package has to add the draw control themselves. The mode can be enabled and disabled programmatically, so the user can choose to hide the default button bar and render their own or use a monkey patch to add a button to the default button bar. The demo uses the monkey patch method. I've updated the readme to include links to information about how to do this, and to the demo code itself. 

I also noticed that the GitHub demo page was non-functional with the changes, so I added the fixes for that and it should continue working if we can get this merged.
